crates/bevy_ui/src/widget/text.rs

```diff
@@ -104,18 +104,18 @@ impl Default for TextNodeFlags {
     Visibility, // TODO: Remove when Node uses required components.
     Transform // TODO: Remove when Node uses required components.
 )]
-pub struct Text(pub String);
+pub struct TextUi(pub String);
```

Comment:
It's good to have consistant naming scheme, but I find this a bit ugly. I'd prefer if we had `Text` refer to UI text and just  add the -`2d` suffix to the text2d types.

Reply:
It's like the core controversy in this PR
I think it's clearer if we have 2d/Ui
also helps with the TextReader situation, where we need to add "Ui" to prevent collision with the generic type

Reply:
I have a strong preference for just Text, and I think Cart will share that.